static/app/components/tooltip.tsx

```diff
@@ -114,74 +111,14 @@ function isOverflown(el: Element): boolean {
   return el.scrollWidth > el.clientWidth || Array.from(el.children).some(isOverflown);
 }
 
-function getTooltipPortal(useGlobalPortal: boolean): HTMLElement {
-  // Use a single global portal element for tooltips
-  if (useGlobalPortal) {
-    let portal = document.getElementById('tooltip-portal');
-    if (!portal) {
-      portal = document.createElement('div');
-      portal.setAttribute('id', 'tooltip-portal');
-      document.body.appendChild(portal);
-    }
-    return portal;
-  }
-
-  // Each tooltip gets it's own div element
+function createTooltipPortal(): HTMLElement {
   const portal = document.createElement('div');
   document.body.appendChild(portal);
   return portal;
 }
 
-/**
- * Codesplit loading of the TooltipStore used for visual acceptance tests
- */
-async function getAcceptanceTestTooltipStore() {
-  const {tooltipStore} = await import('sentry/stores/tooltipStore');
-  return tooltipStore;
-}
-
-type UseTooltipInAcceptanceTestsOptions = {
-  ignore: boolean;
-  setOpen: (open: boolean) => void;
-  setUseGlobalPortal: (useGlobalPortal: boolean) => void;
-  tooltipId: string;
-};
-
-/**
- * When running the acceptance tests, we expose functionality to open / close
- * all of the tooltips. This hook handles registration of tooltips in the store,
- * with special care to unregister tooltips if they become ignored.
- */
-function useTooltipInAcceptanceTests({
-  tooltipId,
-  setOpen,
-  setUseGlobalPortal,
-  ignore,
-}: UseTooltipInAcceptanceTestsOptions) {
-  if (!IS_ACCEPTANCE_TEST) {
-    return;
-  }
-
-  async function registerTooltip() {
-    const tooltipStore = await getAcceptanceTestTooltipStore();
-    tooltipStore?.addTooltip(tooltipId, {setOpen, setUseGlobalPortal});
-  }
-
-  async function unregisterTooltip() {
-    const tooltipStore = await getAcceptanceTestTooltipStore();
-    tooltipStore?.removeTooltip(tooltipId);
-  }
-
-  // Register and unregister tooltips from the store
-  useEffect(() => {
-    if (!ignore) {
-      registerTooltip();
-    }
-    return () => void unregisterTooltip();
-  }, [ignore]);
-}
-
-function Tooltip({
+// Warning: This component is conditionally exported eof based on IS_ACCEPTANCE_TEST env variable
```

Comment:
```suggestion
// Warning: This component is conditionally exported end-of-file based on IS_ACCEPTANCE_TEST env variable
```

had to google what this means 😄